docs/docs/about/why-cli.mdx

```diff
@@ -2,8 +2,62 @@
+
+## Why not another approach or a different CLI?
+
+### Why not just do things manually?
+
+Using CLI for Microsoft 365 is not about replacing the UI, it's about making your life easier. If you need to do a repetitive task, like creating a new SharePoint list every time a new project is started, you can create a script and automate this task with CLI for Microsoft 365. This way you can save time and avoid human errors.
+
+### Why not just write your own CLI or some custom console app tailored to your needs?
+
+You could, but why reinvent the wheel? CLI for Microsoft 365 is a powerful tool that is being actively developed and maintained by a team of experts. It's open-source and free to use. It's also cross-platform and works on any device. It's a great tool that can help you manage your Microsoft 365 tenant more efficiently.
+
+### Why not use SharePoint Online Management Shell?
+
+SharePoint Online Management Shell is a great tool, but it's limited to SharePoint Online and only works with Windows PowerShell. Additionally, it only supports SharePoint administration tasks, so you can only use it to run admin-related commands. In contrast, CLI for Microsoft 365 offers a variety of commands from many different areas of Microsoft 365. It's a more comprehensive tool that can help you manage your Microsoft 365 tenant more efficiently.
+
+### Why not Microsoft Graph CLI or Microsoft Graph PowerShell SDK?
+
+Microsoft Graph CLI and Microsoft Graph PowerShell SDK are amazing, but they are limited to Microsoft Graph API. CLI for Microsoft 365 integrates not only Microsoft Graph but also SharePoint (REST API and CSOM), Azure, Planner, Power Platform, and other APIs. It's a more comprehensive tool that can help you manage your Microsoft 365 tenant more efficiently. Each command documentation is 'handcrafted' not generated code which makes it more human-readable and easier to understand. Last but not least, CLI for Microsoft 365 has commands which are scenario-based and are not just wrappers around Graph API.
+
+### Why not use PnP PowerShell instead?
+
+If you use PowerShell, PnP PowerShell is a convenient solution that you can use to manage different parts of Microsoft 365. CLI for Microsoft 365 offers additional flexibility with the ability to run it in any shell. CLI for Microsoft 365 also offers unique commands to manage SPFx solutions. Whether you're a beginner or an advanced user, and want to use CLI for daily interactive use or scripting, you can configure CLI for Microsoft 365 to your needs making an invaluable tool in your toolbox. CLI for Microsoft 365- offers interactive mode by default which will actively prompt you for required options. CLI for Microsoft 365 is also more intuitive since all commands are grouped into areas like `spo` for SharePoint Online or `entra` for Entra ID, which makes it easier to find the command or operation you are looking for.
```

Comment:
```suggestion
If you use PowerShell, PnP PowerShell is a convenient solution that you can use to manage different parts of Microsoft 365. CLI for Microsoft 365 offers additional flexibility with the ability to run it in any shell. CLI for Microsoft 365 also offers unique commands to manage SPFx solutions. Whether you're a beginner or an advanced user, and want to use CLI for daily interactive use or scripting, you can configure CLI for Microsoft 365 to your needs making an invaluable tool in your toolbox. CLI for Microsoft 365 offers interactive mode by default which will actively prompt you for required options. CLI for Microsoft 365 is also more intuitive since all commands are grouped into areas like `spo` for SharePoint Online or `entra` for Entra ID, which makes it easier to find the command or operation you are looking for.
```